### ISSUE TITLE
Androidacy integration? => Uninstall!

### ISSUE THREAD

**Issue by danielmmmm** [Fox2Code/FoxMagiskModuleManager#210]
Magisk modules are very powerful and can wreak havoc ... if they please. The integration of Androidacy destroys all trust in fmmm. There is no point in fmmm being opensource, when Androidacy Magisk modules cannot be trusted. They are in it for the money, and they modify modules to add advertisements. Who is to say they won't add adware into modules some day? Yes, fmmm would probably remove the Androidacy repo if that were to happen. But ... adware (with system rights!) might already be on my phone by then.
And no, disabling the Androidacy repo is not enough to gain my trust.

Feature request: Get rid of Androidacy and start a collaboration with https://github.com/Magisk-Modules-Repo.

**Comment by HuskyDG** [comment]
Wait for @androidacy-user

**Comment by Fox2Code** [comment]
`Magisk-Modules-Repo` is dead, and when Androidacy repo is disabled there should be no request to it.

@danielmmmm if you find requests to Androidacy while the repo is disabled I'm ready to give you money.

**Comment by danielmmmm** [comment]
> `Magisk-Modules-Repo` is dead
Why are modules there still getting updated? It might be dying, but it doesn't look dead. Support the official repo in fmmm and prevent it from dying 😉

> if you find requests to Androidacy while the repo is disabled I'm ready to give you money.
By then, my phone might already be infected with adware. So, thanks but no thanks.

**Comment by danielmmmm** [comment]
@Fox2Code Nice to see who holds the power here: androidacy-user closing this issue, not you.

Again: Androidacy in your app? => No trust!

Take this not as a request to drop Androidacy, but as a reason some users do not support fmmm.

**Comment by danielmmmm** [comment]
> On a side note, we're very tired of hearing complaints about advertising. Nothing in life is truly free, ...

Yeah, life is hard. Do you share your profits with the module authors? How much, as a percentage of ad revenue, do they get?
Did you @androidacy-user ask module authors for their consent, not only to host their module, but also to add ads to them? We know of at least one case on XDA where you obviously did not ask for consent.

And no, I won't stop arguing against people who misappropriate other peoples work against their will to make money 😘
You closing these issues and shutting down a discussion only shows how little you can be trusted with power. Talking Magisk, we are talking "system rights power".

**Comment by danielmmmm** [comment]
> You don't like advertising, I get it. Use something else then, we're hardly the only solution out there.
I do. Without shedding a tear. I just wanted to let @Fox2Code know.

**Comment by wefalltomorrow** [comment]
> And no, I won't stop arguing against people who misappropriate other peoples work against their will to make money 😘 You closing these issues and shutting down a discussion only shows how little you can be trusted with power. Talking Magisk, we are talking "system rights power".

Completely agree, Androidacy is a cancer to the FOSS Android community. Their attitude is disgusting.
